tools/loader/Loader.cpp

```diff
@@ -271,6 +280,43 @@ static Kinded::Kind getKindFromNodeName(llvm::StringRef nodeName) {
   GLOW_UNREACHABLE("Unknown node name.");
 }
 
+/// Helper to get the BackendKind type from a backend's \p name. Need to be
```

Comment:
It looks like details of the hardware leak into the loader. This means that every time we add a new hardware backend we would need to update this file. Is there a way to avoid it?

@gcatron was working on something simiar.

Reply:
I had a chat with @rdzhabarov last week related to this. When we move to dynamic backend registration there will be a string:backendKind map that can be used. I think this solution is a temporary one until we have that map.

Reply:
We already essentially have this map via `llvm::cl::opt<BackendKind> ExecutionBackend` here in the Loader. If we must have a map like this too, I think we should unify them somehow.

Reply:
I think we can have a follow up PR for this, since some tests command line maybe changed as well.